scalableminds/webknossos
# Do not show task modals in read-only tasks

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a task with taskType A.
- Open a task with taskType B - **a modal informing of the new task type should appear**
- Then finish the task with taskType B and copy the Task ID.
- Open the Tasks view and search for the Task ID. In the results view, click the `+` icon to view the individual task instances.
- You should see your finished task instance, select Actions - View.
- In the opened tracing view **no modals should appear**.

### Issues:
- fixes #3700

------
